zyte_common_items/items.py

```diff
@@ -1202,3 +1204,26 @@ class SocialMediaPost(Item):
     metadata: Optional[SocialMediaPostMetadata] = attrs.field(
         default=None, converter=attrs.converters.optional(MetadataCaster(SocialMediaPostMetadata)), kw_only=True  # type: ignore
     )
+
+
+@attrs.define(kw_only=True)
+class RequestTemplate:
+    filters: Dict[str, Callable]
```

Comment:
I don’t like this, but I cannot think of a better approach to enable custom filtering functions for templates, which might be necessary to handle corner cases.

Reply:
What do you think about waiting, until we get a real-world use case for this? Then we can decide how best to handle it. Having a dict of callables in the item looks wrong, it'd complicate serialization, caching, etc.